src/main/java/com/epam/ms/repository/domain/Event.java

```diff
@@ -0,0 +1,13 @@
+package com.epam.ms.repository.domain;
+
+import lombok.Data;
+
+import java.io.Serializable;
+import java.util.Map;
+
+@Data
+public class Event implements Serializable {
```

Comment:
you don't need to implement `Serializable` interface because you don't use Java built-in mechanism of object serialisation. In order to send data between the systems you're using Jackson which converts objects into the JSON format.
First, java has a very poor performance of serialisation mechanism.
Second, it only allows you to send objects between two JVMs but usually we send information to a 3rd party systems which could be running on a different platform.

Reply:
Thank you very much for the clarification!